app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -108,7 +108,7 @@ def finalize
         # given a composite version, and a list of new components, calculate the list of all components for the new version
         def calculate_components(old_version, new_components)
           old_components = old_version.components.select do |component|
-            !old_version.components.map(&:content_view_id).include?(component.content_view_id)
+            !new_components.map(&:content_view_id).include?(component.content_view_id)
           end
```

Comment:
@bbuckingham this is the change from what we just discovered.  If you want me to walk you through what this is doing and explain why it was wrong, i'd be happy to!